src/python/dxpy/templating/templates/nextflow/dxapp.json

```
@@ -42,6 +42,27 @@
       "help": "Shows additional information in the job logs. If true, the execution log messages from Nextflow will also be included.",
       "class": "boolean",
       "default": false
+    },
+    {
+      "name": "resume",
+      "label": "Resume",
+      "help": "Enables resume functionality in Nextflow workflows. If true, will resume the previous session using the ID declared as resume_session.",
```

I'd say pipelines to distinguish from our DNAnexus workflows.

```suggestion
      "help": "Enables resume functionality in Nextflow pipelines. If true, will resume the previous session using the ID declared as resume_session.",
```

---

Once we have it defined, I'd add a sentence like: "If the Session ID is not provided, the latest job with the properties defined in the Readme is used for resume" (or similar). It can be added to this help or the help for resume_session.